hub/saml-soap-proxy/src/main/java/uk/gov/ida/hub/samlsoapproxy/SamlSoapProxyConfiguration.java

```diff
@@ -88,6 +88,7 @@ protected SamlSoapProxyConfiguration() {
     protected ClientTrustStoreConfiguration rpTrustStoreConfiguration;
 
     @Valid
+    @NotNull
```

Comment:
I don't see eventEmitterConfiguration in SAML SOAP Proxy configuration file in production. Will Hub release automatically add eventEmitterConfiguration to the configuration file in all environments before it starts SAML SOAP Proxy applications?